app.js

```diff
@@ -3,6 +3,14 @@ const cors = require('cors');
 
 require('dotenv').config();
 
+const loginRouter = require('./routes/login');
+const dashboardRouter = require('./routes/dashboard');
+const settingsRouter = require('./routes/settings');
+const teachersRouter = require('./routes/teachers');
+const schoolsRouter = require('./routes/schools');
+const schoolDistrictsRouter = require('./routes/schoolDistricts');
```

Comment:
Rename variables/routes with "schools" to "sites" and "schoolDistricts" to "areas". Update the corresponding file names if needed